src/lib/y2ntp_client/widgets/main_widgets.rb

```diff
@@ -24,13 +24,11 @@ def label
       end
 
       def help
-        # TRANSLATORS: 'man 8 netconfig' is a command, do not translate that
-        _(
-          "The NTP configuration may be provided by the local network over DHCP. " \
+        # TRANSLATORS: configuration source combo box help
```

Comment:
NP: would be nice describe the `%{manual}` placeholder.
```suggestion
        # TRANSLATORS: configuration source combo box help, %{manual} is a manual page reference, e.g. "man 8 netconfig"
```